### PR TITLE
[image] Added screenshot to README

### PR DESCRIPTION
**Pull request by phyxkal**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -207,6 +207,13 @@ Run tests with:
 
     ./runtests.py
 
+Visualizer
+----------
+
+.. figure:: src/image/visual-topology-network.PNG
```
Comment (nemesisdesign) [inline README.rst]: Please move the image to this path: `docs/images/visualizer.png` (including lowercase PNG for consistency with the rest of the openwisp conventions)

File: README.rst [hunk 2]
```diff
@@ -207,6 +207,13 @@ Run tests with:
 
     ./runtests.py
 
+Visualizer
+----------
```
Comment (nemesisdesign) [inline README.rst]: Remove this heading and then move the image to the top, after the following line of text:

`OpenWISP 2 network topology module (built using Python and Django web-framework).`

Comment (phyxkal) [reply README.rst]: after or before the table of content?

Comment (phyxkal) [reply README.rst]: owh i get it

**Comment by phyxkal** [comment]
thx

**Pull request merged**